packages/visx-xychart/test/components/Axis.test.tsx

```diff
@@ -33,70 +34,44 @@ describe('<AnimatedAxis />', () => {
     expect(AnimatedAxis).toBeDefined();
   });
   it('should render a VxAnimatedAxis', () => {
-    expect(setup(<AnimatedAxis orientation="bottom" />).find(VxAnimatedAxis)).toHaveLength(1);
+    expect(
+      setup(<AnimatedAxis orientation="bottom" />).container.querySelectorAll('.visx-axis'),
+    ).toHaveLength(1);
   });
 });
 
-const AxisComponent = (_: unknown) => null;
-
 describe('<BaseAxis />', () => {
   it('should be defined', () => {
     expect(BaseAxis).toBeDefined();
   });
-  it('should use scale=xScale for orientation=top/bottom', () => {
```

Comment:
hi @williaster, this is tricky to do within RTL because it is checking on function `xScale` being passed down through props and this counts as a checking on implementation detail. After consideration, I am thinking of removing this test for now... Do you think this test is necessary?

Reply:
ill give that a shot.
I did `const xScale ={ domain: jest.fn(()=>[0, 1, 2]), range: jest.fn(() => [0, 10]) };`
it complains about missing `tickPosition` function, i don't know where that one is being passed from

Reply:
That actually works! please check my changes at [L56](https://github.com/gazcn007/visx/pull/1/files#diff-4c14754ad489f7c017cdcbed31a18c7f890c5e7048ad4e7428b278a2634f7f46R56)

🚀